.github/workflows/tests.yaml

```diff
@@ -40,15 +41,18 @@ jobs:
         env:
             BUILD_VARIANT: ${{matrix.build_variant}}
             CHIP_TOOL_VARIANT: ${{matrix.chip_tool}}
-            TSAN_OPTIONS: "halt_on_error=1 suppressions=scripts/tests/chiptest/tsan-linux-suppressions.txt"
+            TSAN_OPTIONS:
```

Comment:
Whatever auto-formatter is being used here disagrees with previous auto-formatters.  Please undo all the unrelated changes.